onadata/apps/logger/models/submission_review.py

```diff
@@ -51,6 +55,19 @@ class Meta:
         """
         app_label = 'logger'
 
+    @classmethod
+    def set_deleted_at(cls, review_id, deleted_at=timezone.now(), user=None):
+        """
+        Retrieves the submission_review object and calls
+        set_deleted with it
+        """
+        try:
+            submission_review = cls.objects.get(id=review_id)
```

Comment:
This `cls.objects` and `cls.DoesNotExist` feels very weird to me.  I'd vote that you just use the model class name as that is how these statements are usually presented